test/torch_np/numpy_tests/linalg/test_linalg.py

```diff
@@ -802,7 +841,7 @@ def test_singular(self):
         for A, p in itertools.product(As, p_neg):
             linalg.cond(A, p)
 
-    @xfail  # (
+    @skip(reason="NPY_VER: fails on CI")  # (
```

Comment:
some say NPY_VER, other NP_VER and others just fail on CI. Let's just use one of them so that we can consistently grep for all of them